store/tikv/lock_resolver.go

```diff
@@ -573,3 +576,47 @@ func (lr *LockResolver) resolveLock(bo *Backoffer, l *Lock, status TxnStatus, cl
 		return nil
 	}
 }
+
+func (lr *LockResolver) resolvePessimisticLock(bo *Backoffer, l *Lock, cleanRegions map[RegionVerID]struct{}) error {
+	tikvLockResolverCountWithResolveLocks.Inc()
+	for {
+		loc, err := lr.store.GetRegionCache().LocateKey(bo, l.Key)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		if _, ok := cleanRegions[loc.Region]; ok {
```

Comment:
maybe we should remove this in next pr.

it seems txn first resolve a optimistic lock then can not resolve pessimistic lock, if mix optimistic/pessmistic